spateo/tools/cluster_degs.py

```diff
@@ -11,13 +11,11 @@
 from scipy.stats import mannwhitneyu
 from sklearn.neighbors import NearestNeighbors
 from statsmodels.sandbox.stats.multicomp import multipletests
+from sympy import N
```

Comment:
did you used sympy later? also you may need to add sympy into the requirements.txt file. Alternatively, you will need to import it only in the function you used it and report an error when the users don't install it. something like this: 
    try:
        import sympy as N
    except ImportError:
        raise ImportError("You need to install the package `sympy`." "install sympy via `pip install sympy`")